service/discovery/discovery_test.go

```diff
@@ -452,8 +506,8 @@ func (m mockDiscoverer) List() ([]voc.IsCloudResource, error) {
 	}
 }
 
-func (mockDiscoverer) CloudServiceID() string {
-	return testutil.TestCloudService1
+func (d *mockDiscoverer) CloudServiceID() string {
```

Comment:
Do you really need the method?

Reply:
Yes, the `discovery.Discoverer` interface needs it.